src/command.cpp

```diff
@@ -48,6 +48,57 @@ const char* Command::getstring() const
 }
 
 // add opcode
+bool Command::checkError(int64_t& e, JSON& json, ErrorDetails& errorDetails)
```

Comment:
Maybe `e` --> `errorCode` will be more meaningful

Reply:
I've used `int64_t` because some commands return a integer value instead of an error. Then I think it's better return a generic type and after that convert to type